sky/shell/ui/engine.h

```diff
@@ -94,6 +96,8 @@ class Engine : public UIDelegate,
   gfx::Size physical_size_;
   std::string initial_route_;
   blink::SkyDisplayMetrics display_metrics_;
+  std::string language_code_;
+  std::string country_code_;
```

Comment:
It's sort of lame that we need to cache these values both here and in SkyView.  Given that we only ever call sky_view_->SetLocale after calling CreateView, maybe we can remove the caching from SkyView.  (We should be able to remove the SkyDisplayMetrics caching in the same way.